nixos/modules/services/misc/remote-builder.nix

```diff
@@ -0,0 +1,133 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  mkRemoteBuilderDesc = config:
```

Comment:
This doesn't need to be a function, because the only `config` used is already in scope, even with the same name.